apps/ledger-live-mobile/src/components/SelectFeesStrategy.tsx

```diff
@@ -67,17 +70,18 @@ export default function SelectFeesStrategy({
   const closeNetworkFeeHelpModal = () => setNetworkFeeHelpOpened(false);
 
   const onPressStrategySelect = useCallback(
-    (item: any) => {
+    (item: FeeStrategy) => {
       onStrategySelect({
         amount: item.amount,
-        label: item.forceValueLabel ?? item.label,
-        userGasLimit: item.userGasLimit,
```

Comment:
@haammar-ledger could you later study why we have a generic component here where it definitely sounds like this is Ethereum specific? would help on the typing here, but we can't do proper typing if it's not inside families/ethereum/*

Reply:
why --> historical :sweat_smile: 
It's actually used by both [ethereum](https://github.com/LedgerHQ/ledger-live/blob/e2da2d229c83a9a65fbdeafcf86ec183614d4eba/apps/ledger-live-mobile/src/families/ethereum/EthereumFeesStrategy.tsx#L9) and [bitcoin](https://github.com/LedgerHQ/ledger-live/blob/e2da2d229c83a9a65fbdeafcf86ec183614d4eba/apps/ledger-live-mobile/src/families/bitcoin/SendRowsFee.tsx#L9).
Probably made sense when there were only a couple coins supported, but maybe not worth it aymore.